examples/demo-stf/src/runtime.rs

```diff
@@ -36,6 +38,7 @@ use sov_modules_macros::{DispatchCall, Genesis, MessageCodec};
 /// Similar mechanism works for queries with the difference that queries are submitted by users directly to the rollup node
 /// instead of going through the DA layer.
 
+#[cfg_attr(feature = "native", cmd(DefaultContext))]
```

Comment:
Can we rename the cmd macro to something that makes its functionality more clear?